lib/widgets/button.dart

```diff
@@ -94,6 +95,8 @@ enum ViamButtonSizeClass {
   /// The font size of the button, based on size class
   double get fontSize {
     switch (this) {
+      case xxs:
+        return 10;
```

Comment:
[q] why the same font size as `xs`?

Reply:
Anything smaller than 10 becomes pretty difficult to read